schema/job.ts

```diff
@@ -0,0 +1,31 @@
+import z from "zod";
+
+export const saveJobsSchema = z.object({
+  companyName: z
+    .string()
+    .min(1, "Company name should contain atleast 1 character")
+    .max(50, "Company name should contain atmost 50 characters"),
+  jobTitle: z
+    .string()
+    .min(3, "Job title should contain atleast 3 character")
+    .max(50, "Job title should contain atmost 50 characters"),
+  jobDescription: z
+    .string()
+    .max(200, "Job Description should contain atmost 200 characters")
```

Comment:
We should have a min character count on this too. Without a job description the post will be pretty bad. Say 100 characters. you can max it at 2000.

Reply:
Done ✅